pkg/rpcserver/rpcserver.go

```diff
@@ -105,7 +106,7 @@ func New(cfg *mgrconfig.Config, mgr Manager, debug bool) (*Server, error) {
 	if !cfg.Experimental.RemoteCover {
 		features &= ^flatrpc.FeatureExtraCoverage
 	}
-	return newImpl(&Config{
+	return newImpl(context.Background(), &Config{
```

Comment:
Thinking if context should be part of config. We already have it in the local config. The config is supposed to provide all required parameters.
What's you reasoning to pass it separately?

Reply:
I'd say all of the Go library methods, which I've seen use context, accept it as the first parameter. On the other hand, though, none of them takes a `Config` struct..

I don't have a strong opinion on this.